docs/netbsd.md

```diff
@@ -2,49 +2,75 @@
 
 ## How to run syzkaller on NetBSD using qemu
 
-So far the process is tested only on linux/amd64 host. To build Go binaries do:
+
+--->>>Steps to set up NetBSD on qemu: 
```

Comment:
Re "--->>>", please stick to official markdown syntax.
See https://guides.github.com/features/mastering-markdown/ for reference.
I think we either need to use subsections (`###`), or lists and sub-lists.
The same applies to all uses of `--->>>` below.

You can always preview the rendered markdown after committing the change to your fork, e.g. for this pull request:
https://github.com/google/syzkaller/pull/535/commits/09fd1ca49a9e8276395561653b98fd42475d3f11?short_path=be906ad#diff-be906ade9a9ac48742ecfc3c2a763872

Reply:
"--->>>" Are changed to ordered list as suggested by link.

Reply:
Humm... I still see "--->>>".
Did you forget to upload a new version?